river_dl/rnns.py

```diff
@@ -47,39 +69,52 @@ def train_step(self, data):
         with tf.GradientTape(persistent=True) as tape:
             y_pred = self(x, training=True)  # forward pass
 
-            loss_main = nnse_one_var_samplewise(y, y_pred, 0)
-            loss_aux = nnse_one_var_samplewise(y, y_pred, 1)
+            loss_main = nnse_one_var_samplewise(y, y_pred, 0, self.tasks)
+            if self.tasks == 2: 
+                loss_aux = nnse_one_var_samplewise(y, y_pred, 1, 2)
```

Comment:
```suggestion
                loss_aux = nnse_one_var_samplewise(y, y_pred, 1, self.tasks)
```
even though we know that `self.tasks == 2` when we reach this line, it's still clearer to use the variable name